src/platforms/common/data-management/sensitive-data/index.mdx

```diff
@@ -66,23 +66,15 @@ For more details, see [_Filtering Events_](../../configuration/filtering/).
 
 Instead of sending confidential information in plaintext, considering hashing it:
 
-```javascript
-Sentry.setTag("birthday", checksum_or_hash("08/12/1990"));
-```
+<PlatformContent includePath="sensitive-data/set-tag" />
```

Comment:
here we could not reuse the already defined `set-tag` cus this example has specific PII data and the existent `set-tag` don't